wiki/Jobs.md

```diff
@@ -85,3 +85,50 @@ First create a project. A project is a collection of jobs. Then click the add ic
     ]
 }
 ```
+
+### Example with pull request in checkout.
+
```

Comment:
Also, I think you should get rid of this section. Its really redundant. And the parameter is already documented elsewhere.